sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/BlobsCheckpointStoreTests.cs

```diff
@@ -60,7 +60,7 @@ public async Task ListCheckpointsAsync_LogsOnInvalidCheckpoints()
                     {
                         Page<BlobItem>.FromValues(new[]
                         {
-                            BlobsModelFactory.BlobItem("testnamespace/testeventhubname/testconsumergroup/checkpoint/0", false, BlobsModelFactory.BlobItemProperties(false), metadata: new Dictionary<string, string>())
+                            BlobsModelFactory.BlobItem("testnamespace/testeventhubname/testconsumergroup/checkpoint/0", false, BlobsModelFactory.BlobItemProperties(false, contentLength: 0), metadata: new Dictionary<string, string>())
```

Comment:
Why do pass contentLength of 0?

Reply:
Because there is no content for a checkpoint blob; its a zero-byte file.  The data exists entirely in the metadata.   I'm not sure why the text is explicitly setting this, as no logic relies on it, but it does mirror what is expected for the actual service return.

Reply:
There is logic to skip empty blobs at https://github.com/azure/azure-sdk-for-net/blob/b19a863bf1b55df9a9c62773fbef643d76fab4b8/sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/BlobCheckpointStore/BlobsCheckpointStore.cs#L354

Reply:
ah, fair enough.   I was thinking in terms of the T2 checkpoints.   Makes sense in the legacy context.